daemon/container.go

```diff
@@ -117,7 +117,7 @@ func (daemon *Daemon) Register(c *container.Container) error {
 	defer c.Unlock()
 
 	daemon.containers.Add(c.ID, c)
-	return c.CheckpointTo(daemon.containersReplica)
+	return c.CheckpointTo(context.TODO(), daemon.containersReplica)
```

Comment:
Same here; it's good to set the `ctx := context.TODO()` at the start of the function